src/pages/TokenDetails/Widget.tsx

```diff
@@ -0,0 +1,183 @@
+import { TransactionReceipt } from '@ethersproject/abstract-provider'
```

Comment:
thoughts on putting the Widget in a higher level file since theoretically it could be used outside of token details?